pp.c

```diff
@@ -771,7 +770,7 @@ S_do_chomp(pTHX_ SV *retval, SV *sv, bool chomping)
                                is the buffer inside PL_rs, so don't free it.
                              */
                             assert (temp_buffer == rsptr);
-                            goto nope_free_sv;
+                            goto nope_free_nothing;
```

Comment:
I don't think this line belongs in this PR.  I haven't worked it out, but the comments seem to me to indicate that the current code is wrong, and the change is correct.  But I don't understand fully.

The rest of the PR is fine

Reply:
He removed svrecode, so nope_free_sv no longer makes sense (I think) as it seems to exist only to free svrecode. Am I missing something?

Reply:
Yes, that was the rationale and I thought that compilers might warn about an unused `nope_free_sv`.

FWIW `svrevcode` did used to get used, but that stopped being the case with https://github.com/Perl/perl5/commit/8df0e7a28b22521824d9a5d0d44f2b27d2955298

Reply:
Nod. I think its fine to merge this. 

Yves